### PR TITLE
Add redis-interval-module

### PR DESCRIPTION
**Pull request by ogama**
Add the redis-interval-module repository in modules.json

### CODE REVIEW

**Review by zuiderkwast — APPROVED** [review]
Nice!

For reference, this module started after this discussion: https://github.com/redis/redis/issues/8368

File: modules.json [hunk 1]
```diff
@@ -386,6 +386,15 @@
...
     ],
     "stars":0
-  }
-
+  },
+  {
+    "name": "redis-interval-module",
+    "license": "MIT",
+    "repository": "https://github.com/ogama/redis-interval-module",
+    "description": "Redis module for creation and manipulation of interval set.",
```
Comment (madolson) [inline modules.json:394]: interval sets*

**Comment by zuiderkwast** [comment]
You decided to use `] ... [` for open intervals, which is fine, but typically Redis commands use `( ... )` for open intervals. Some commands like [`XRANGE`](https://redis.io/commands/xrange) use `(` as a prefix to a number to indicate an open interval.

I just wanted to point this out. I will still merge this PR. :-)

File: modules.json (hunk 1 above)
Comment (zuiderkwast) [reply modules.json:394]: I agree, but it's not very important, so I decided to merge as-is.

Comment (madolson) [reply modules.json:394]: Sure, I agree

**Pull request merged**